Newsletter/Makefile

```diff
@@ -0,0 +1,22 @@
+TEXOPTS= --lualatex \
```

Comment:
instead of these options, just pass `-r ../common/latemkrc` to latexmk, see e.g.:

https://github.com/pep-dortmund/latex-documents/blob/b6b9786ffbc6f78a7919ccbee921d91885442112/absolventenfeier/Makefile#L14-L15

Reply:
Thanks, will do